aip/general/0121.md

```diff
@@ -53,6 +53,19 @@ database schema. In fact, having an API that is identical to the underlying
 database schema is actually an anti-pattern, as it tightly couples the surface
 to the underlying system.
 
+### Resource requirements
+
+For a valid resource-oriented API, the following must be satisfied:
+
+- Each resource **must** have a unique [resource name][] by which it can be
+  identified and referenced.
+- The resource API must support [Get][].
+- The resource API must support [List][], with the exception of [singleton
+  resources][].
+- Resources must operate over a stateless protocol: any individual request is
```

Comment:
Can we just say "in a stateless manor"? A protocol is like `tcp` or `http` afaict and the word, doesn't quite fit here.